lib/commons/builder/queries/information_from_positions.rq.liquid

```diff
@@ -0,0 +1,33 @@
+SELECT DISTINCT
+  ?position {% lang_select 'position_name' %}
+  ?positionType
+  ?adminAreaTypes
+  ?adminArea {% lang_select 'admin_area' %}
+  ?positionSuperclass {% lang_select 'position_superclass' %}
+  ?legislature {% lang_select 'legislature' %}
```

Comment:
The `?legislature` variable also gets executives bound to it. Should it be renamed to `?body` instead?